README.md

```diff
@@ -26,34 +26,47 @@
 
 **nf-core/references** is a bioinformatics pipeline that build references, for multiple use cases.
 
-It is primarily designed to build references for common organisms and store it on [AWS iGenomes](https://github.com/ewels/AWS-iGenomes/).
+For most common organisms the plan is to build the references and store them in the cloud, similar to [AWS iGenomes](https://github.com/ewels/AWS-iGenomes/).
 
-From a fasta file, it will be able to build the following references:
+![nf-core/references metro map](docs/images/nf-core-references_metro_map_color.png)
+
+## DNASEQ
+
+For DNASEQ pipelines (ie [nf-core/sarek](https://nf-co.re/sarek)), needing only a fasta file, it will be able to build the following references:
 
-- Bowtie1 index
-- Bowtie2 index
 - BWA-MEM index
 - BWA-MEM2 index
 - DRAGMAP hashtable
 - Fasta dictionary (with GATK4)
 - Fasta fai (with SAMtools)
-- Fasta sizes (with SAMtools)
 - Fasta intervals bed (with GATK4)
 - MSIsensor-pro list
+- SNAP aligner index
+
+It will compress VCF files, if it was not already compressed, and tabix index it.
```

Comment:
What has VCF files got to do with FASTA references?

Reply:
Still confused because the intro sentence of the whole thing only refers to references :sweat_smile: 

But too late now :grimacing:

Reply:
I'll rephrase it later, no worries

Reply:
I know what I meant, and I just need to explain it more properly